pkg/oam/env.go

```diff
@@ -34,11 +35,18 @@ func GetEnvByName(name string) (*types.EnvMeta, error) {
 	return &meta, nil
 }
 
+//Create or update env.
+//If it does not exist, create it and switch to the new env.
+//If it exists, update it and switch to the new env.
 func CreateOrUpdateEnv(ctx context.Context, c client.Client, envName string, namespace string) (error, string) {
```

Comment:
- I'm confused with func names  between `CreateEnv` and `UpdateEnv`, and `CreateOrUpdateEnv`.
- API and Cli should depend the same function in oam package, but `CreateOrUpdateEnv` only used by API.

Reply:
API and Cli use different user habits, so I suggest that cli still retains the ability to switch while creating

Reply:
@wonderflow Please also take a look at this design.